geowebcache/wms/src/main/java/org/geowebcache/config/GetCapabilitiesConfiguration.java

```diff
@@ -89,6 +90,8 @@
 
     private String vendorParameters = null;
 
+    private String cachedParameters = null;
```

Comment:
Maybe give it a collection of parameter filters instead of a String.  This fixes the problem with values containing `&` or `=` and with leaving all of the parameters completely open.